index.html

```diff
@@ -0,0 +1,161 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <link rel="stylesheet" href="./css/mainstyles.css">
+    <link rel="stylesheet" href="./css/tabletstyles.css">
+    <link rel="stylesheet" href="./css/mobilecss.css">
+    <title>shelter</title>
+</head>
+
+<body>
+    <div class="container">
+        <div class="head">
+            <header>
+                <div class="logo">
+                    <h2 class="title">Cosy House</h2>
+                    <h4 class="subtitle">Shelter for pets in Boston</h4>
+                </div>
+                <ul class="links">
+                    <a class="active" href="">About the shelter</a>
+                    <a href="">Our pets</a>
+                    <a href="">Help the shelter</a>
+                    <a href="">Contacts</a>
+                </ul>
+            </header>
+            <main>
+                <div class="ban-content">
+                    <h1 class="headerh1">Not only people need a house</h1>
+                    <p class="headerp">We offer to give a chance to a little and nice puppy with an extremely wide and open
+                        heart. He or she will love you more than anybody else in the world, you will see!</p>
+                    <button class="button">Make a friend</button>
```

Comment:
```suggestion
                    <button class="button" type="button">Make a friend</button>
```

Кнопкам нужно указывать аттрибут `type`
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/button#attr-type